sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/Commands.scala

```diff
@@ -23,15 +27,15 @@ object Commands {
     val processIO = BasicIO.standard(false).withOutput(outputCapture).withError(toErrorLog)
     val code: Int = process.run(processIO).exitValue()
     if (code != 0) {
-      sys.error(s"Non-zero exit code: $code")
+      Left(s"Non-zero exit code: $code")
```

Comment:
This change let's the output parser execute even when `webpack` fails. The call to `sys.error` was too early in this case